release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md

```diff
@@ -76,9 +76,87 @@ To view bug fixes, see the [GitHub milestone for 2201.2.0 (Swan Lake)](https://g
 
 ### Runtime updates
 
+#### New features
+
+##### Strand dump
+
+Allows getting the status of strands and strand groups during the execution of a Ballerina program.
+
+This can be used to troubleshoot runtime errors. The Ballerina runtime will emit the strand dump to the standard output stream in the text format, if it receives a `SIGTRAP` signal (`SIGTRAP` is not available on Windows).
+
+E.g., if the PID of the running Ballerina program is `$PID`, you can get the strand dump by executing either `kill -SIGTRAP $PID` or `kill -5 $PID` command.
+
+##### `StopHandler` Object
+
+Allows registering a function that will be called during graceful shutdown.
+
+A call to `onGracefulStop` will result in one call to the handler function that was passed as an argument; the handler functions will be called after calling `gracefulStop` on all registered listeners, in the reverse order of the corresponding calls to `onGracefulStop`.
+
+E.g., a `foo` function can be called during the graceful shutdown by registering it as follows.
+
+`runtime:onGracefulStop(foo);`
+
+#### Improvements
+
+#### New runtime Java APIs
```

Comment:
We do not need this heading

Reply:
I think the heading `New runtime Java APIs` is added according to the release note template. But we can remove it as well.

Reply:
Resolved in the next comment.